### ISSUE TITLE
Hello, there are some questions I would like to ask you.

### ISSUE THREAD

**Issue by qiuxuelaici** [pss1207/sleep_apnea#3]
Hello, I am a student from China, I am fortunate to see your code about sleep_apnea and have learned a lot.
Because I am stupid, there are some places that are not fully understood and I would like to ask you. In pre_proc.py, the data of 35 patients given on physionet is processed and the apn file is used. Then, if we want to test with the model after train.py, how to deal with the test set given on physionet (x01-x35), because the test set is not given an apn file.
I have just been in touch with GitHub, and I am very grateful to you for being a predecessor with unselfish open source code. At the same time, I don't know how to ask questions. If my questioning method or the question itself is wrong, please forgive me.

**Comment by pss1207** [comment]
Hi, thank you for your email.
No problem for any question. I am busy on weekdays. Can I give you an
answer on weekend? I need to review my code. It’s been a long time since I
uploaded my code :)

Best Regards,
Sangshik

**Comment by qiuxuelaici** [comment]
Wow, I can't believe I got your reply so soon. My question is not urgent, you can use your free time to answer it. 
Thanks again for your reply.

Best Regards,
jie

**Comment by pss1207** [comment]
Hi Jie,
I couldn't get test data so I used 10% of the training data for the test.
But I didn't separate test patients and this is not the correct way.
I think I have to update the code for training and test.
Thank you!

**Comment by pss1207** [comment]
> Wow, I can't believe I got your reply so soon. My question is not urgent, you can use your free time to answer it.
> Thanks again for your reply.
> 
> Best Regards,
> jie

Hi Jie,
I couldn't get test data so I used 10% of the training data for the test.
But I didn't separate test patients and this is not the correct way.
I think I have to update the code for training and test.
Thank you!

**Comment by qiuxuelaici** [comment]
Hi,Sangshik,
I have seen your updated code. I need to spend some time learning it and hope to continue discussions with you later. Thank you again for your answer.
